rental.rb

```diff
@@ -0,0 +1,10 @@
+class Rental
+  attr_accessor :date
+  attr_reader :person, :book
```

Comment:
- [ ] Kindly add `:person` and `:book` to `attr_accessor` since user can read and write them. See suggestion below:

```suggestion
  attr_accessor :date, :person, :book
```